ts/global/modal.ts

```diff
@@ -1,5 +1,5 @@
 import kebabCase from 'lodash.kebabcase'
 
-window.$openModal = name => {
-  return window.dispatchEvent(new Event(`open-modal:${kebabCase(name)}`))
+window.$openWireModal = name => {
```

Comment:
Why need to rename this function? I search in the other package and I don't see a function with the same name